bibliopixel/remote/trigger_process.py

```diff
@@ -0,0 +1,10 @@
+from .. util import importer
```

Comment:
:) Yeah... Windows kicks my ass here. I'd love to drop it from support but that would kinda defeat the point of Python I guess :P